build.sbt

```diff
@@ -483,3 +509,39 @@ lazy val testingCompiler =
       )
     )
     .dependsOn(testingCompilerInterface, nativelib)
+
+lazy val testInterface =
+  project
+    .settings(toolSettings)
+    .settings(scalaVersion := libScalaVersion)
+    .in(file("test-interface"))
+    .settings(
+      name := "test-interface",
+      libraryDependencies += "org.scala-sbt"    % "test-interface"   % "1.0",
+      libraryDependencies -= "org.scala-native" %%% "test-interface" % version.value % Test
+    )
+    .enablePlugins(ScalaNativePlugin)
+    .dependsOn(testInterfaceSerialization, stubs)
```

Comment:
Do we really need to split test interface and stubs sub projects? Can we merge them together?